Robinhood/Robinhood.py

```diff
@@ -128,24 +176,21 @@ def login(self,
         if mfa_code:
             payload['mfa_code'] = mfa_code
 
-        try:
-            res = self.session.post(self.endpoints['login'], data=payload)
-            res.raise_for_status()
-            data = res.json()
-        except requests.exceptions.HTTPError:
-            raise RH_exception.LoginFailed()
+        res = self.session.post(self.endpoints['login'], data=payload)
+        self.raiseForStatus(res)
```

Comment:
This couple's hard with another PR.  Merge should be able to handle this.

Also if that PR changes significantly, this PR will need adjustment.

Last, we remove the chance to raise a library-specific exception here that should be useful